### ISSUE TITLE
Behaviour of super key changed(On fresh install as well)

### ISSUE THREAD

**Issue by Rayvert64** [pop-os/pop#2473]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Gnome-Shell

**Issue/Bug Description:**
Basically, I used to be able to change workspaces by pressing the `super` key and then scroll the mouse wheel. Now instead of that, nothing happens and after releasing the super key, I find myself in the "gnome overview."

**Steps to reproduce (if you know):**
1. press `super` + scroll wheel(up or down)
2. release `super`
3. frustration

**Expected behavior:**
The combination of `super` and scrolling the mouses wheel, should change desktops. On release of the `super` key nothing should happen.

**Comment by Rayvert64** [comment]
Note: this happened on an updated system AND a system with a fresh install. Yesterday(June 13th) this bug was not present.

**Comment by Rayvert64** [comment]
![image](https://user-images.githubusercontent.com/15572474/173655724-dc140458-4b0a-40b0-a48d-3d2e75b7fa8b.png)

**Comment by linuxgnuru** [comment]
The super key can be set to either start the launcher or show workspaces and can be changed in `Settings -> Desktop -> Desktop Options` If you set it to open Workspaces the mouse wheel should let you scroll workspaces.
